src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -511,6 +511,9 @@ namespace SimAirServingZones {
 			return;
 		}
 
+		// initialize on off fan PLR to 1 for those systems that do not set this variable
+		DataAirLoop::LoopOnOffFanPartLoadRatio = 1.0;
+
```

Comment:
This needs to change. What if there are multiple air loops with multiple OA systems. The OA systems are all simulated at the same time and will all use this same variable. This needs to be an array LoopOnOffFanPartLoadRatio( NumAirloops).